mkdocs.yml

```diff
@@ -103,37 +102,40 @@ markdown_extensions:
 # Page tree
 nav:
   - Overview:
-    - Introduction: index.md
-    - Community: community.md
-    - Conceptual: conceptual.md
+      - Introduction: index.md
```

Comment:
What happened here?

Reply:
good question, again I think the vscode linter did something